README.md

```diff
@@ -1,70 +1,181 @@
 
-## Learn More
+To get a local copy up and running, follow these steps.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Setup
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Clone this repository to your desired folder:
 
-### Code Splitting
+> cd my-folder
+> git clone https://github.com/jubaer919/react-group-project.git
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Prerequisites
 
-### Analyzing the Bundle Size
+In order to run this project you need:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- gitHub account;
+- git installed on your OS.
 
-### Making a Progressive Web App
+### Install
+> https://github.com/microverseinc/linters-config/tree/master/html-css-js
+- Installations required to run this project:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Webhint installation. 
+- Run the following command:
+> npm install --save-dev hint@7.x
 
-### Advanced Configuration
+### Stylelint installation.
+- Run the following command:
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### ESLint
+- Run
+
+> npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+### Usage
+
+You can use this project by cloning it to your folder and changing index.html and styles.css files.
+
+<p align="right">(<a href="#usage">back to top</a>)</p>
+
+### Run tests
+
+To run tests, run the following commands:
+
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint ."
+```
+To Open The Project On The Browser
+npm start
+
+<p align="right">(<a href="#run-tests">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can redeploy this project by adding new lines of code to source files.
+
+<p align="right">(<a href="#deployment">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jubaer hossain maruf**
+
+- GitHub:[jubaer919] (https://github.com/jubaer919)
```

Comment:
- [ ] This lie is not rendering as expected, it important to have a professional Readme kindly fix it by adding @ at the beginning of the name:

GitHub:[@jubaer919] (https://github.com/jubaer919)